src/module/system/action-macros/helpers.ts

```diff
@@ -120,6 +112,18 @@ export class ActionMacroHelpers {
             ].flat();
             const selfActor = actor.getContextualClone(combinedOptions.filter((o) => o.startsWith("self:")));
 
+            let title = "";
+            if (options.actionGlyph) {
+                title += `<span class="pf2-icon">${options.actionGlyph}</span> `;
+            }
+            title += `<b>${game.i18n.localize(options.title)}</b>`;
+            const subtitle =
+                typeof options.subtitle === "function"
+                    ? options.subtitle?.({ actor: selfActor, target: targetActor })
+                    : options.subtitle;
+            title += ` <p class="compact-text">(${game.i18n.localize(subtitle)})</p>`;
+            const content = (await options.content?.(title)) ?? title;
+
```

Comment:
- Prefer IIFE over `let`
- Prefer `<strong>` over `<b>`
- You touched it, so now it has to go in a template or at least be cobbled together via DOM API :slightly_smiling_face: